core/src/main/java/hudson/triggers/SlowTriggerAdminMonitor.java

```diff
@@ -28,7 +30,8 @@
     @NonNull
     private final Map<String, Value> errors = new ConcurrentHashMap<>();
 
-    public static int MAX_ENTRIES = 10;
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
+    public static /* Script Console modifiable */ int MAX_ENTRIES = SystemProperties.getInteger(SlowTriggerAdminMonitor.class.getName() + ".maxEntries", 10);
```

Comment:
Needs corresponding update to https://www.jenkins.io/doc/book/managing/system-properties/ and perhaps even separate changelog entry.